docs/DeveloperGuide.md

```diff
@@ -965,7 +965,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1. User requests to sort the task list by date.
 2. Coach2k22 shows the updated task list, sorted with the tasks with the earliest deadline at the top.
-
+    Use case ends.
```

Comment:
```suggestion

    Use case ends.
```

Reply:
I meant to include have a new line before `Use Case ends.` (make sure to have the format be the same as the previous use cases).